packages/vapor/resources/icons/svg/arrow-top-slim.svg

```diff
@@ -3,7 +3,7 @@
 <svg version="1.1" id="Calque_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
 	 viewBox="0 0 29 43" style="enable-background:new 0 0 29 43;" xml:space="preserve">
 <style type="text/css">
-	.st0{fill:none;stroke-width:3;stroke-linecap:round;stroke-linejoin:round;}
+	.st0{stroke-width:3;stroke-linecap:round;stroke-linejoin:round;}
 </style>
 <path class="st0" stroke="#282829" d="M14.4,41.2v-39"/>
 <path class="st0" stroke="#282829" d="M2.1,15.7L14.4,2.2l12.3,13.5"/>
```

Comment:
Doesn't seem to be right:

![image](https://user-images.githubusercontent.com/260007/135093644-54767507-ba8a-4284-8fe8-0e74ec971e80.png)

```suggestion
	.svgArrowTopSlimPaths{fill:none;stroke-width:3;stroke-linecap:round;stroke-linejoin:round;}
</style>
<path class="svgArrowTopSlimPaths" stroke="#282829" d="M14.4,41.2v-39"/>
<path class="svgArrowTopSlimPaths" stroke="#282829" d="M2.1,15.7L14.4,2.2l12.3,13.5"/>
```

Reply:
Sorry I missed that. Thanks for pointing that out!